.schemas/config.schema.json

```diff
@@ -651,6 +651,30 @@
               }
             }
           ]
+        },
+        "timeout": {
```

Comment:
I think it would be better to  nest those under (and update their names):

```
retry:
  # Sets the connection timeout. If reached, the request will canceled be retried.
  conn_timeout: 100ms

  # The request fails after the specified amount of time was reached and no success response was received. 
  max_wait: 5s
```

Reply:
I made these changes and fixed the test, but CI is still failing for some reason